stl/inc/mutex

```diff
@@ -463,7 +463,7 @@ public:
     }
 
     ~scoped_lock() noexcept {
-        _For_each_tuple_element(_MyMutexes, [](auto& _Mutex) noexcept { _Mutex.unlock(); });
+        _STD apply([](_Mutexes&... _Mtxes) { (..., void(_Mtxes.unlock())); }, _MyMutexes);

```

Comment:
We conventionally write `(void)meow` instead of `void(meow)`:

```suggestion
        _STD apply([](_Mutexes&... _Mtxes) { (..., (void) _Mtxes.unlock()); }, _MyMutexes);
```